pkg/agent/openflow/pipeline.go

```diff
@@ -785,8 +794,7 @@ func (c *client) l3FwdFlowToPod(localGatewayMAC net.HardwareAddr, podInterfaceIP
 		flows = append(flows, flowBuilder.MatchDstIP(ip).
 			Action().SetSrcMAC(localGatewayMAC).
 			Action().SetDstMAC(podInterfaceMAC).
-			Action().DecTTL().
-			Action().GotoTable(l3FwdTable.GetNext()).
+			Action().GotoTable(l3DecTTLTable).
```

Comment:
@wenyingd : I made this change, because later with SNAT policy, we can have packets from gw0 which will hit this flow as well after un-SNAT'd (such packets' dst MAC will be rewritten to vMAC in conntrackStateTable), but we want not to decrement TTL for them.
Also we can keep all routed packets (handled by l3Fwd funcs) consistent.

Let me know what you think.

Reply:
Yes, I hope to use L3DecTTLTable on both source and destination port, for the reasons I mentioned above.

> if a packet is sending from local Pod to gateway port would also go into L3DecTTLTable (e.g. in NoEncap mode)
I think in that case the host stack should decrement the TTL? So, in l3FwdFlowRouteToGW() I skip L3DecTTLTable().